static/src/javascripts/projects/commercial/modules/third-party-tags/inizio.js

```diff
@@ -0,0 +1,34 @@
+// @flow strict
+import config from 'lib/config';
+
+const onLoad = () => {
+    const handleQuerySurveyDone = (
+        surveyAvailable: boolean,
+        survey: { measurementId: string }
+    ) => {
+        if (surveyAvailable) {
+            if (window && window.googletag) {
+                window.googletag.cmd.push(() => {
+                    window.googletag.pubads().setTargeting('inizio', 't');
+                });
+            }
+            console.log(`surveyAvailable: ${survey.measurementId}`);
```

Comment:
helpful while we are testing the end-to-end implementation, can be removed once testing completed